packages/shared/components/Logo.svelte

```diff
@@ -1,47 +1,40 @@
 <script lang="ts">
     import { appSettings, appStage } from '@core/app'
-    import { get } from 'svelte/store'
+    import { Logo as LogoEnum, LogoOverrideStage } from './enums'
 
-    export let logo = undefined
-    export let width = undefined
-    export let height = undefined
-    export let classes = ''
-    export let overrideStage: undefined | 'alpha' | 'beta' | 'prod' = undefined
-
-    $: darkModeEnabled = $appSettings.darkMode
-    $: selected = logos[logo]?.[overrideStage ?? get(appStage)]
+    export let logo: LogoEnum
+    export let width: string = '100%'
+    export let height: string = '100%'
+    export let overrideStage: undefined | LogoOverrideStage = undefined
```

Comment:
We could think about this prop here, im not sure I understand what it means by reading it, do you know what it is used for? Could we maybe think it to be a bit more intuitive?